packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
-|Success  |Green (#92D400/ #486b00)   |fa-check-circle | Use to indicate that a task or process has completed successfully
-|Default  |Cyan (#73C5C5/ #004368)   |fa-bell | Use for generic messages with no associated severity
 
 ## Usage
-There are two primary use cases when alerts are used:
+Alerts can be used to communicate:
 
-* **Synchronous notifications:** Synchronous notifications are returned as a direct response to some user action like clicking the Submit button on a form.
+* **Synchronous notifications:** Returned as a direct response to some user action like clicking the Submit button on a form.
 
-* **Asynchronous notifications:** Asynchronous notifications are generated by the system in response to some internal event. They may reflect status of a process, an internal error, or similar system event.
+* **Asynchronous notifications:** Generated by the system in response to some internal event. They may reflect status of a process, an internal error, or similar system event.
 
-Three alert patterns are defined in PatternFly for addressing these use cases.
+There are three ways that the system can communicate notifications to the users.
 
 |           |Usage     |UI placement     |Persistence
 |-----------|----------|-----------------|---------------|
-|[Inline alerts](#inline-alerts)|Synchronous notifications that are a direct response to a user action, eg., clicking the Submit button, and can be informational or can identify actions that are required on that page before the user can continue. |Appear at the top of the content area and push other content down. | Persist until the user dismisses them or navigates away from the page.
-|[Toast alerts](#toast-alerts) |A system notification that doesn’t interrupt the user’s current workflow. |Enter from the right edge of the screen and overlay content. |Remain until they time out, or the user dismisses them.
-|[Modal alerts](#modal-alerts) |Interrupting the flow until the user takes action. |Overlay the UI and prevent further user actions until the user closes the modal. |Yes, until the user dismisses the modal.
+|[Inline alerts](#inline-alerts)|Use for notifications that are a direct response to a user action, eg., clicking the Submit button, and can be informational or that identify actions required on that page before the user can continue. |Appear at the top of the content area and push other content down. | Persist until the user dismisses them or navigates away from the page.
+|[Toast alerts](#toast-alerts) |Use to present a system notification that should not interrupt the user’s current workflow. |Enter from the right edge of the screen and overlay content. |Remain until they time out, or the user dismisses them.
+|[Modal alerts](#modal-alerts) |Use when you must interrupt the task at hand until the user takes action. |Overlay the UI and prevent further user actions until the user closes the modal. |Remain until the user dismisses the modal.
```

Comment:
I wonder if it makes sense to include modal alerts in this doc because they are technically not related to the component we call "Alerts", and the link does not link to anything...

Reply:
That's a fair point @mmenestr . I had originally had more content about modals here but took it out for reasons you said. If I remove this, do you think I need the table above or is this redundant with what's said above? Maybe just combine/condense this text?

Reply:
Yes I think combining and condensing the text, and getting rid of the table would be a good call here!